app/components/common/paperItem/__tests__/__snapshots__/index_spec.tsx.snap

```diff
@@ -7,32 +7,6 @@ exports[`PaperItem Component when paper data doesn't have DOI should render corr
   <div
     className="contentSection"
   >
-    <div>
```

Comment:
Please fill the mock data to show proper rendering result.
(unser `__mocks__` directory, maybe `paper.json`)